app/scripts/satisfaction.js

```diff
@@ -1,11 +1,20 @@
-window.onload = function () {
+jQuery(document).ready(function($) {
```

Comment:
I need all the things wrapped in this (document).ready function to happen before the things in the window.load function.  I made it less ugly for you now.